composer.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "spryker-eco/heidelpay",
+  "description": "Heidelpay bundle",
+  "license": "proprietary",
+  "require": {
+    "heidelpay/php-api": "dev-master",
+    "heidelpay/php-customer-messages": "dev-master",
+    "spryker/checkout": "^3.0.0",
+    "spryker/currency": "^2.0.0 || ^3.0.0",
+    "spryker/kernel": "^3.0.0",
+    "spryker/locale": "^3.0.0",
+    "spryker/money": "^2.0.0",
+    "spryker/oms": "^6.0.0 || ^7.0.0",
+    "spryker/payment": "^3.0.0 || ^4.0.0",
+    "spryker/propel-orm": "*",
+    "spryker/quote": "^1.0.0",
+    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0",
+    "spryker/session": "^3.0.0",
+    "spryker/step-engine": "^2.0.0 || ^3.0.0",
+    "spryker/symfony": "^3.0.0",
+    "spryker/util-encoding": "^2.0.0",
+    "spryker/zed-request": "^3.0.0"
+  },
+  "require-dev": {
+    "spryker/code-sniffer": "*",
+    "spryker/country": "*",
+    "spryker/customer": "*",
+    "spryker/propel": "*",
+    "spryker/testify": "*"
+  },
+  "suggest": {
+    "spryker/silex": "*",
+    "spryker/installer": "If you want to use Installer plugins you need to install spryker/installer."
+  },
+  "autoload": {
+    "psr-4": {
+      "SprykerEco\\": "src/SprykerEco/",
+      "SprykerEcoTest\\": "tests/SprykerEcoTest/"
```

Comment:
This is a code smell that should be removed before releasing, those should go into autoload-dev.